packages/dds/sequence/src/intervalCollection.ts

```diff
@@ -304,7 +304,7 @@ export class LocalIntervalCollection<TInterval extends ISerializableInterval> {
 						"Adding an interval that belongs to another interval collection is not permitted",
 					);
 				}
-				interval.addProperties(props);
+				interval.properties = addProperties(interval.properties, props);
```

Comment:
Is there a signature of `addProperties` that takes more than one `PropertySet` argument?

Reply:
no currently, but the method is internal, so we can modify it if needed

Reply:
Is this code expected to work then? Or do you plan to modify it as a part of this task?